### DISCUSSION TITLE
How to get pixel location of detected bounding box?

### DISCUSSION THREAD

**Discussion by DhiviJR** [mindee/doctr#1773] [Q&A] [answered]
### 🚀 The feature

How can I get the array of detected Bounding box location from the image or pdf?

### Motivation, pitch

To get the array of detected Bounding box location from the image or pdf?

**Comment by felixdittrich92** [comment, marked as answer]
Hi @DhiviJR :wave:

You can export the results take a look here: https://mindee.github.io/doctr/using_doctr/using_models.html#what-should-i-do-with-the-output

The geometry values are absolute (0 -1) so you need to multiply the x-coordinates with the doc/image width and the y-coordinates with the image height to get the relative coords :)

Best,
Felix

**Reply by DhiviJR** [reply]
Thank you for your guidance. It is helpful.

**Comment by felixdittrich92** [comment]
If you need only the detection part without recognition check out our notebooks :) --> https://mindee.github.io/doctr/notebooks.html    (standalone predictors)
